numpy/core/tests/test_ufunc.py

```diff
@@ -941,6 +946,27 @@ def test_identityless_reduction_noncontig_unaligned(self):
         a = a[1:, 1:, 1:]
         self.check_identityless_reduction(a)
 
+    def test_initializer_reduction(self):
+        # np.minimum.reduce is an identityless reduction
+
+        # For cases like np.maximum(np.abs(...), initializer=0)
+        # More generally, a supremum over non-negative numbers.
+        assert_equal(np.maximum.reduce([], initializer=0), 0)
+
+        # For cases like reduction of an empty array over the reals.
+        assert_equal(np.minimum.reduce([], initializer=np.inf), np.inf)
+        assert_equal(np.maximum.reduce([], initializer=-np.inf), -np.inf)
+
+        # Rendom tests
+        assert_equal(np.minimum.reduce([5], initializer=4), 4)
+        assert_equal(np.maximum.reduce([4], initializer=5), 5)
```

Comment:
Can you add the other two tests with `minimum` and `maximum` swapped?

Reply:
Done!

Reply:
Added!